<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckDiary_Dialog</class>
 <widget class="QDialog" name="CheckDiary_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ascend_pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时间升序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="descend_pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时间降序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="undone_pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>未完成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="done_pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>已完成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="query_pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>186</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="in_pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>入库药品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="out_pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>出库药品</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>23</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>75</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>127</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="yyyy_spinBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QSpinBox" name="mm_spinBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>72</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QSpinBox" name="dd_spinBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>123</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>452</width>
     <height>352</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="remindDate_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>218</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>282</y>
     <width>131</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>322</y>
     <width>131</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>362</y>
     <width>131</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>0</y>
     <width>8</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/backGround (4).jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>0</y>
     <width>8</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>114</y>
     <width>131</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>63</y>
     <width>131</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>31</y>
     <width>469</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>0</y>
     <width>8</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>593</x>
     <y>31</y>
     <width>8</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>393</y>
     <width>461</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>ascend_pushButton</zorder>
  <zorder>descend_pushButton</zorder>
  <zorder>undone_pushButton</zorder>
  <zorder>done_pushButton</zorder>
  <zorder>query_pushButton</zorder>
  <zorder>in_pushButton</zorder>
  <zorder>out_pushButton</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>yyyy_spinBox</zorder>
  <zorder>mm_spinBox</zorder>
  <zorder>dd_spinBox</zorder>
  <zorder>tableWidget</zorder>
  <zorder>remindDate_label</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_8</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
